Followup cleanup of docs about added features for UniFi Protect (#20990)
home-assistant/home-assistant.github.io: source/_integrations/unifiprotect.markdown

    * CONTROLLER PERMISSIONS - Under UniFi Protect, select Administrators. **NOTE**: Other roles may work, but only the default Administrators rule is fully tested.
1. Click *Add* in at the bottom Right.

<div class='note'>

> A few users have reported that they had to restart their UDMP device after creating the local user for it to work. So if you get some kind of *Error 500* when setting up the Integration, try restart the UDMP.

Reviewer: ```suggestion
A few users have reported that they had to restart their UDMP device after creating the local user for it to work. So if you get some kind of *Error 500* when setting up the Integration, try to restart the UDMP.
```

> A few users have reported that they had to restart their UDMP device after creating the local user for it to work. So if you get some kind of *Error 500* when setting up the Integration, try to restart the UDMP.

Reviewer: What about the UNVR/UNVRPROs?

Author: That is a good question. I also do not know how important that is anymore since we have made a lot of changes to the config flow. Maybe better to just leave to the troubleshoot tip in the bottom and remove this?

Reviewer: Since they are all technically NVRs we could say NVR.  It seems like it belongs in troubleshooting though as you suggest.

Author: Remove it and added a note about 502s as well since I have been seeing those lately with UFP going down.

</div>

Each UniFi Protect camera will get a device in Home Assistant with the following:

* **Camera** - A camera for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
* **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](/integrations/tts/#service-say).
* **Privacy Mode** - If your camera allows for Privacy Masks, there will be a configuration switch to toggle a "Privacy Mode" that disables recording, microphone, and a black privacy zone over the whole camera.
* **Device Configuration** - Cameras will get various different configuration controls based on the features available to the camera. Currently provided configuration controls:
  * configuration sliders for Chime Duration, Zoom Level, Microphone Sensitivity, and WDR Level
  * configuration switches Overlay Information, Smart Detections types, Status Light, HDR, High FPS mode, System Sounds.

> * **Button** - A disabled by default button is added for each camera device. The button will let you reboot your camera device.

Author: @frenck How does this look? I refactor it out to not lay out every individual entity, but instead highlight important controls/sensors as well as an overview of the provided configuration entities.

Button is at the bottom for all of them and separate because it is a "control" (so important), but still disabled by default, disabled by default configuration entities (i.e. the SSH toggle) are not even mentioned.

Also, explicitly laid out a "UniFi Protect device" vs a HA entity so the terminology should hopefully be less confusing ("smart sensor" for the UFP device vs "sensor" for the HA entity).

* **Privacy Mode** - If your camera allows for Privacy Masks, there will be a configuration switch to toggle a "Privacy Mode" that disables recording, microphone, and a black privacy zone over the whole camera.
* **Sensors** - Sensors include "Is Dark", "Motion Detected", "Detected Object" (if the camera supports smart detections), and "Doorbell Chime" (if the camera has a chime). Several diagnostics sensors are added including sensors on uptime, network connection stats, and storage stats. Doorbells will also have a "Voltage" sensor for troubleshooting electrical issues.
  * Motion and Detected Object sensors will also contain an `event_thumbnail` attribute that can be used to get the event thumbnail directly from UniFi Protect for that motion or smart detection.
  * The Detected Object sensor will have the values of "none", "person", or "vehicle" based on object UniFi Protect detected.

> * **Device Configuration** - Cameras will get various different configuration controls based on the features available to the camera. Currently provided configuration controls:

Reviewer: ```suggestion
* **Device Configuration** - Cameras will get various configuration controls based on the features available to the camera. Currently provided configuration controls:
```

  * configuration sliders for Chime Duration, Zoom Level, Microphone Sensitivity, and WDR Level
  * configuration switches Overlay Information, Smart Detections types, Status Light, HDR, High FPS mode, System Sounds.
* **Button** - A disabled by default button is added for each camera device. The button will let you reboot your camera device.

### UniFi Protect Floodlights

Each UniFi Protect floodlight will get a device in Home Assistant with the following:

* **Light** - A light entity will be added for each floodlight device. The light entity will let you control turning on or off your light as well as adjust the brightness of your floodlight.
* **Sensors** - Sensors are provided for data provided from floodlight devices. Sensors include "Is Dark" and "Motion Detected".

> * **Device Configuration** - Floodlights will get configuration controls for the PIR Motion Sensitivity, the Auto-shutoff Duration after detected motion, Status Light switch and SSH status.

Reviewer: ```suggestion
* **Device Configuration** - Floodlights will get configuration controls for the PIR Motion Sensitivity, the Auto-shutoff Duration after detected motion, Status Light switch, and SSH status.
```

### Enabling Debug Logging

> Both the UniFi Protect integration and the Python library it uses provides a ton of debug logging that can help you with troubleshooting connectivity issues. To enable debug logging for both, add the following to your `configuration.yaml` file:

Reviewer: ```suggestion
Both the UniFi Protect integration and the Python library it uses provide debug logging that can help you with troubleshooting connectivity issues. To enable debug logging for both, add the following to your `configuration.yaml` file:
```
